### PR TITLE
5651 Update metadata

### PR DESCRIPTION
**Pull request by LeoFuna**
### Ticket(s)

- _[[Airtable ticket 5651](https://airtable.com/appfJZShN8K4tcWHU/pagxblGyhI5EJIA3v?HjEAY=recP087J5ZRwGItAX)]_

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Description

Meta tags were not working as expected by the implementation.
Need to find a way, using Next page route, to make meta tags work properly.
Found a possible solution at: [Next Doc](https://nextjs.org/docs/pages/api-reference/components/head)

#### Documentation possible solution
image

### Checklist:

- [x] I have performed a self-review of my own code
- [x] There are no new linting errors/problems in my code
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

**Comment by e-papanicolas** [comment]
@LeoFuna Good catch! I read the doc you linked, and I am thinking that our tags are also not in line with the bottom guideline, that all tags must be direct children of the head. I wonder if additionally, the entire social meta component should be abandoned, and the meta tags added directly to `_document.tsx`.

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
@LeoFuna Good catch! I read the doc you linked, and I am thinking that our tags are also not in line with the bottom guideline, that all tags must be direct children of the head. I wonder if additionally, the entire social meta component should be abandoned, and the meta tags added directly to `_document.tsx`.

This guideline also explains why google analytics hasn't been working since it is much more nested, so that is a great find.

**Comment by e-papanicolas** [comment]
Can you also move the IndividualPageHead component in `index.tsx` to the top. The google analytics div will need to be fixed as well, but I am not sure if it being above the <Head> is interfering in some way.

File: components/helper/IndividualPageHead.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function IndividualPageHead({ title, metaContent }: Props) {
   return (
     <Head>
       <title>{title}</title>
-      <meta name="description" content={metaContent} />
+      <meta name="description" content={metaContent} key="description" />
```
Comment (e-papanicolas) [inline components/helper/IndividualPageHead.tsx:13]: On the line above, should `<title>` be `<meta title="">` ?

Comment (LeoFuna) [reply components/helper/IndividualPageHead.tsx:13]: Im not sure if we really need to change the `title` one.

If I understood the docs right, the `key` property is just necessary when we have meta tags that are used multiple times, so its to not duplicate, but to overwrite. 

And I didnt find any `title` tag used in `_document.tsx`. What do you think?

Comment (e-papanicolas) [reply components/helper/IndividualPageHead.tsx:13]: @LeoFuna Hm I'm not sure honestly. Looking at the documentation you linked, the first example shows just the `<title>` but the second example also has a meta tag just below it. I'll leave it to you to choose, and we can always readjust if needed.

Comment (LeoFuna) [reply components/helper/IndividualPageHead.tsx:13]: Ok so I will try without changing the `title` just to not change something that might not be the problem.
And if that dont fix what we want, we can readjust as you said.

**Review by e-papanicolas — APPROVED** [review]
@LeoFuna awesome job on this!

**Pull request merged**